docs/maven-plugin.md

```diff
@@ -103,6 +105,8 @@ Parameters:
 | **maven.checkstyle.version** | String | The version of the maven-checkstyle-plugin that will be used (default value is **2.17**)|
 | **checkstylePlugins** | List<Dependency> | A list with artifacts that contain additional checks for Checkstyle |
 | **checkstyleProperties** | String | Relative path of the properties file to use in the ruleset to configure specific checks |
+|**skipCheckstyle**|boolean|If set to true no checkstyle checks will be executed.|
+|**skippedCheckTypes**|Collection<String>|A collection of all the check types that will be skipped during the execution of checkstyle.If not set all checks are executed.

```

Comment:
To me this is not clear what these types are, while reviewing I realised it might be filetypes, is that so, should we spell that out here instead of check types?

Reply:
Yes, they are the file types that the check processes. I added this to the docs.